Xamarin.Forms.Platform.Android/Renderers/PickerRenderer.cs

```diff
@@ -110,6 +110,9 @@ void IPickerRenderer.OnClick()
 		{
 			Picker model = Element;
 
+			if (_dialog != null)
```

Comment:
To be honest I just looked at how it was fixed in the AppCompat variant and implemented that here :)

They do the same thing there, just assumed that there would be a reason

Reply:
Haha easy peasy! :)

Reply:
I would like to add something besides this condition _dialog.IsShowing check this will ensure that the dialog already displayed

Reply:
Is there any specific reason that you are requesting this? I see that it is an extra check and it makes sense, but I don't see how this could go wrong. It has been like this for ages in the AppCompat library and never seen any issues with it